core/openapi/schemas/jobCollection.yaml

```diff
@@ -1,3 +1,9 @@
-    type: array
-    items:
-      type: string
+jobCollection:
```

Comment:
Remove top level element

Reply:
Dear Benjamin, 
I have removed the top level elements.

You can find an exemple of additionalParameters defined for the input identifier "a"  from [here](https://demo.mapmint.com/WPS3/processes/HelloPy2).